src/util/ChartUtils.ts

```diff
@@ -250,7 +252,10 @@ export const getBarSizeList = ({
         result[cateId].push({
           item: barItems[0],
           stackList: barItems.slice(1),
-          barSize: isNil(selfSize) ? globalSize : selfSize,
+          barSize:
+            isNil(selfSize) && isNil(globalSize)
```

Comment:
nit: can store `isNil(selfSize)` in a variable above this to prevent calling twice